launcher/FileSystem.cpp

```diff
@@ -174,36 +176,57 @@ bool copy::operator()(const QString& offset)
     auto src = PathCombine(m_src.absolutePath(), offset);
     auto dst = PathCombine(m_dst.absolutePath(), offset);
 
-    std::error_code err;
+    std::error_code err{};
 
     fs::copy_options opt = copy_opts::none;
 
     // The default behavior is to follow symlinks
     if (!m_followSymlinks)
         opt |= copy_opts::copy_symlinks;
 
+    const auto testAndCopy = [opt, &err](const QString& s, const QString& d) {
+        if (ensureFilePathExists(d)) {
+            fs::copy(toStdString(s), toStdString(d), opt, err);
```

Comment:
I'm pretty sure this can be done cleaner with Qt

Reply:
I decided to keep this as the function was changed to use std's filesystem 5932f36285ebf3ec7d852db1b80630589224eccc. I'm not too familiar with Qt, so I wouldn't know if there's an idiomatic way for this, but I can change it to Qt only if that's what you want.

Reply:
I'm not the final authority anymore, but I trust what worked for (presumably) years more than what some random dude changed.

Reply:
Also; I can help you with converting to Qt if that's what we plan to do

Reply:
> I'm not the final authority anymore, but I trust what worked for (presumably) years more than what some random dude changed.

fs::copy was what was used for years though? So you trust fs::copy then? Or are you referencing something else?